### ISSUE TITLE
Folders show sum of child counts, not actual number of unique items

### ISSUE THREAD

**Issue by ghost** [lwindolf/liferea#1283]
When the same item appears in multiple feeds in the same folder, viewing the folder shows only one of each item in the list, but the item count shown for the folder in the tree is the sum of counts of all feeds.

**Comment by lwindolf** [comment]
That's true. At the same time unread count aggregation is one of the most performance costly things in Liferea so it is good to avoid doing more complex calculations (e.g. duplicate elimination).

**Comment by lwindolf** [comment]
Closing this as not planned.
